Client/Frontend/AuthenticationManager/AuthenticationSettingsViewController.swift

```diff
@@ -115,28 +115,32 @@ extension RequirePasscodeSetting: PasscodeEntryDelegate {
     }
 }
 
-class TouchIDSetting: Setting {
-    private let authInfo: AuthenticationKeychainInfo?
+class AuthenticationSetting: Setting {
+    private var authInfo: AuthenticationKeychainInfo?
 
     private weak var navigationController: UINavigationController?
     private weak var switchControl: UISwitch?
 
-    private var touchIDSuccess: (() -> Void)? = nil
-    private var touchIDFallback: (() -> Void)? = nil
+    private var touchIDSuccess: (AuthenticationSetting -> Void)? = nil
```

Comment:
Nit: One way to avoid having to type `(AuthenicationSetting -> Void)?` each time we want to use/declare a block for authentication is to use `typealias` to name it something more meaningful and less verbose:

```swift
    typealias AuthenticationCallback = (AuthenticationSetting -> Void)?
```

Then you can use that wherever you declare your closures!

```swift
     private var touchIDSuccess: AuthenticationCallback
```